packages/trpc/src/lib/client/client.ts

```diff
@@ -9,8 +14,6 @@ import {
   tRPC_CACHE_STATE,
 } from './cache-state';
 import { createTRPCRxJSProxyClient } from './trpc-rxjs-proxy';
-import { CreateTRPCClientOptions } from '@trpc/client/src/createTRPCUntypedClient';
-import { HTTPHeaders } from '@trpc/client/src/links/types';
```

Comment:
Ummmm yes, this is not intended to be used by anyone. Not sure how to block this